PiwikTracker.php

```diff
@@ -1288,6 +1288,25 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param $proxy
+     * @param int $proxyPort
```

Comment:
Can you maybe add an example and the type for `$proxy`? Just to let everyone easily know in which format something needs to be set. Port is pretty obvious but maybe there could be one or two examples for proxy